### PR TITLE
Add driver driver payload

### PR DESCRIPTION
**Pull request by NoeTerrier**
Add a binary containing a driver that can be used to interract with miralis from S-mode (or U-mode through ioctls).

Refactor Makefile to create every binary with only make command.

### CODE REVIEW

File: .github/workflows/release.yaml
Comment (CharlyCst) [inline .github/workflows/release.yaml:1]: If I understand correctly you create a new binary that uses the driver to print the statistics once Linux is done booting? Then you probably want to publish the new binary in the release too by adding it at the end of this file.

**Review by CharlyCst — COMMENTED** [review]
You might just have forgot to publish the new artifacts in the release, can you check my comment?

File: .github/workflows/release.yaml
Comment (NoeTerrier) [reply .github/workflows/release.yaml:1]: Yep exactly. The driver code could be later updated to handle ioctls or other kinds of operations if wanted to test things.

**Review by CharlyCst — APPROVED** [review]
Looks all good for me :)

You can create a new release, and if the build succeed you can merge this PR yourself.

**Pull request merged**